docs/developers/node-tutorial.md

```diff
@@ -53,14 +53,14 @@ an example public Core Endpoint.
 
 > Note: You are also encouraged to find a community-run API endpoint
   and there are several in the Discord. This one is used for demonstration
-  purposes.
+  purposes. You can find a list of RPC endpoints [here](../nodes/mamaki-testnet#rpc-endpoints)
 
-Here we are starting a light node with a connection to a Core endpoint at
-`https://api-mamaki.pops.one/` and also telling the light node to use the `developer`
-key we generated as its default account.
+Here we are starting a light node with a connection to a Core endpoint and
+also telling the light node to use the `developer` key we generated
+as its default account.
 
 ```sh
-celestia light start --core.grpc https://api-mamaki.pops.one/ --keyring.accname developer
+celestia light start --core.grpc http://<ip-address>:9090 --keyring.accname developer
```

Comment:
Is that port always the same or is configurable?

Reply:
good question, I've seen examples of someone use port 8090 but I think the default is 9090 cc @renaynay @Wondertan @Bidon15

Reply:
9090 is the default port grpc for celestia-app @liamsi 

We can substitute 9090 for <port> here (which I recommend) but also mention in comments that 9090 is the default for celestia-app so it's likely it's open on 9090

Reply:
It is configurable in app.toml
if the app.toml in celestia-app node instance is left as default, then 9090.